### ISSUE TITLE
Partial delete, some items stays and can't be deleted.

### ISSUE THREAD

**Issue by AntoineTurmel** [SpiderOak/Encryptr#247]
I have an issue sometimes when deleting an item : "Error, Container has not changed" with button "Try again".

When I click on an item I have "LOADING..." and it's loading forever, I can't remove it from my account.

(Windows, version 2.0.0)

**Comment by devgeeks** [comment]
:/

I'll see what might be causing it. Maybe when I push 2.0.1 to fix the scrollbars I can have the app clear out entries from the index that don't really exist.

**Comment by Valkoinen** [comment]
I have absolutely the same problem. After "Error. Container has not changed" item will start to load, but never finish. Can't remove it.
Linux Mint 17.2, Encryptr v2.0.0

**Comment by devgeeks** [comment]
Yeah, this need a fix asap. I'll see what I can do.

**Comment by bwafflemaker** [comment]
I've seen the same on iOS, but I also had my vault open on my Ubuntu machine simultaneously.

I'm also seeing the behavior mentioned where a deleted item comes back to the list but never loads if selected.

Thanks, @devgeeks!

**Comment by devgeeks** [comment]
Yeah, I *think* I know what is happening. I'll need to first fix it so it doesn't happen anymore, then make a "migration" script that runs when you upgrade to the fixed version that cleans out any rogue entries.

**Comment by whitedp** [comment]
Just updating: Issue still the same;

> I have the same issue: After deleting an item, in my case a "general" item, it still shows up in the list of items and when I attempt to open it, it hangs, trying to load forever.
> 
>  I don't get the "Error. Container has not changed." mentioned above. I've attached a screenshot of the window as it tries to load the item
> [Screenshot.pdf](https://github.com/SpiderOak/Encryptr/files/392049/Screenshot.pdf)
> 
> I am using version 2.0.0
>

**Comment by AntoineTurmel** [comment]
Any update on this issue ?

**Comment by renegat59** [comment]
Hi, same here. Looking forward to a fix:)

**Comment by whitedp** [comment]
Same here, looking forward to fix :)

**Comment by matknny** [comment]
Hi, I am getting the same problem, I cannot open or delete an entree. Just coming up "Loading".

**Comment by renegat59** [comment]
I was trying to find a and fix the bug, but so far no luck on running the app after clonning from github:P After weekend I may have some more information about this

**Comment by fancsali** [comment]
Sorry ladies and gents, but any fix in sight? I've got an entry that cannot be opened, just loads forever. (Both on Mac and Android)
